### ISSUE TITLE
Experiment results in machine readable format

### ISSUE THREAD

**Issue by ogencoglu** [RUCAIBox/LLMSurvey#48]
Thanks for your work!

Do you have the experiment results in machine readable format (csv, json, xlsx etc.) instead of an image?

**Comment by EliverQ** [comment]
Yes, we are preparing these results in machine readable format. However, this part of the work is quite tedious, so you may need to wait for a while. Thank you for your attention to our survey.
